app/views/digital_collections/_home_showcase.html.erb

```diff
@@ -29,4 +29,10 @@
 
   <% end %>
 
+  <%= content_for :twitter_og_meta do %>
```

Comment:
I think content_for blocks may interfere with (or not get) cached. (I think this partial gets cached?): https://apidock.com/rails/ActionView/Helpers/CaptureHelper/content_for

Reply:
@corylown Ack, good catch. I hadn't thought about that. On that page:
> WARNING: `content_for` is ignored in caches. So you shouldn’t use it for elements that will be fragment cached.

I'll revisit how I set this up to see if there's a good way around that issue.

Reply:
We can also revisit whether this really needs to get cached at all...

Reply:
I like that the showcase area gets cached; it might be relatively resource intensive to render it uncached with all the image-related methods and randomization in the mix. Now that I look closer, it looks like the `<% end %>` right above the `<%= content_for %>` is where the showcase cache block ends so we should be in the clear, at least on this partial.